sdk/src/test/java/com/qonversion/android/sdk/UserPropertiesBuilderTest.kt

```diff
@@ -0,0 +1,142 @@
+package com.qonversion.android.sdk
+
+import com.qonversion.android.sdk.dto.UserProperty
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+
+internal class UserPropertiesBuilderTest {
+
+    private lateinit var builder: UserPropertiesBuilder
+
+    @BeforeEach
+    fun setUp() {
+        builder = UserPropertiesBuilder()
+    }
+
+    @Test
+    fun `set name`() {
+        // given
+        val name = "test name"
+
+        // when
+        builder.setName(name)
+
+        // then
+        assertThat(builder.properties[UserProperty.Name.code]).isEqualTo(name)
```

Comment:
Should we use "_q_name" or UserProperty.Name.code for the key?

Reply:
I guess no, as we don't test concrete enum value code